### DISCUSSION TITLE
Custome fileter like AdGuard MV3

### DISCUSSION THREAD

**Discussion by ethpsantos** [uBlockOrigin/uBOL-home#37] [Ideas]
It is just an ideia and I think it is better to have it documented.

You can add custom filters like in ADGuard MV3.

**Comment by gorhill** [comment]
I can't make it not-lite or more complicated than it is now, I don't have the time to maintain more than I do. uBO Lite will have to stay lite.

**Reply by dportvine** [reply]
Has your opinion not changed? It is important for users from Russia to have **RU AdList: Counters**.

<details>
<summary>Screen</summary>

https://github.com/uBlockOrigin/uBOL-home/assets/111344219/1e961912-2813-4702-9f48-4cff5b3cbc53
</details>

**Reply by stephenhawk8054** [reply]
@dimisa-RUAdList Sorry for the ping, if `RU AdList: Counters` list is included in uBO / uBO Lite but not enabled by default (including Russian-language browsers), would it be OK for you?

**Reply by dimisa-RUAdList** [reply]
@stephenhawk8054 Yes, sure.
